src/BrandEmbassyCodingStandard/Sniffs/Classes/TraitUseSpacingSniff.php

```diff
@@ -28,35 +28,17 @@ final class TraitUseSpacingSniff implements Sniff
     public const CODE_INCORRECT_LINES_COUNT_BETWEEN_USES = 'IncorrectLinesCountBetweenUses';
     public const CODE_INCORRECT_LINES_COUNT_AFTER_LAST_USE = 'IncorrectLinesCountAfterLastUse';
 
-    /**
-     * @var int
-     */
-    public $linesBeforeFollowingPropertyOrConstant;
+    public int $linesBeforeFollowingPropertyOrConstant;
```

Comment:
[m] Pozor, tady je potenciální neinicializovaná public property. Vzhledem k tomu, jak `SniffSettingsHelper::normalizeInteger()` funguje, použil bych nulu.
```suggestion
    public int $linesBeforeFollowingPropertyOrConstant = 0;
```